public/editor/scripts/editor/js/bramble-editor.js

```diff
@@ -14,7 +14,13 @@ define(function(require) {
       // Start loading the Bramble editor resources
       Bramble.load("#webmaker-bramble",{
         url: options.editorUrl,
-        hideUntilReady: true
+        hideUntilReady: true,
+        extensions: {
+          enable: [
+            // Watch and warn users on rename or delete of index.html
+            'bramble-watch-index.html'
```

Comment:
How is the locale being passed down on the top-level URL?  Aren't we doing that here if you request it? https://github.com/mozilla/brackets/blob/master/src/bramble/client/main.js#L344-L346  Maybe we need to set that flag so locale gets included?